tests/Unit/Schema/Directives/RenameDirectiveTest.php

```diff
@@ -44,7 +44,7 @@ public function testThrowsAnExceptionIfNoAttributeDefined(): void
 
         $this->schema = /** @lang GraphQL */ '
         type Query {
-            foo: String! @rename
+            fooBar: String! @rename
```

Comment:
Why?

Reply:
Because the lazy fields have this as a side effect, when fields are not accessed they also won't throw.

This is not an issue with the validation command, that works perfectly but it is an issue when you are not calling the field you are testing 😄